### DISCUSSION TITLE
Section 3_pytorch_computer_vision_video -- What is use of -q and -U during installation of a package?

### DISCUSSION THREAD

**Discussion by Durga-33** [mrdbourke/pytorch-deep-learning#289] [Q&A] [answered]
Video timestamp 19:16:48 

<img width="667" alt="K" src="https://user-images.githubusercontent.com/122006734/217196314-6ef8f00c-2a02-4fb5-b67b-a041ad691485.png">

`!pip install -q torchmetrics -U mlxtend`
In the above code, what does -q and -U stand for? What is their meaning?

Thanks

**Comment by mrdbourke** [comment, marked as answer]
Hi @Durga-33,

Good question!

`-q` stands for "quiet" as in there won't be any outputs when installing the package.

And `-U` stands for "update" as in, get the latest version of the package *if* it's already installed (for example, if package version 1.0.0 is installed but 1.0.2 is available, remove 1.0.0 and install 1.0.2).

See more here: https://pip.pypa.io/en/stable/cli/pip/#general-options

**Reply by Durga-33** [reply]
Thanks a lot, Sir!
